nxrm-aws-ha-helm/values.yaml

```diff
@@ -23,7 +23,15 @@ statefulset:
   container:
     image:
       repository: sonatype/nexus3
-      tag: 3.45.1
+      tag: 3.50.0
+    resources:
+      # See help documentation, these are minimum sysem requirements
```

Comment:
I chose these values based on our minimum requirements in https://help.sonatype.com/repomanager3/product-information/system-requirements